frontend/src/App.tsx

```diff
@@ -196,14 +198,12 @@ export class App extends PureComponent<Props, State> {
 
   private readonly embeddingId: string = generateUID()
 
-  constructor(props: Props) {
+  public constructor(props: Props) {
```

Comment:
There a reason to make this public? Im assuming this works like java where undisclosed means protected / private though.

Reply:
No, the keyword is optional here (I just prefer specificity). In this case, App's constructor _is_ already public because it's a component we instantiate elsewhere. (If you try to make the constructor private, you'll get a typing error.)